### ISSUE TITLE
FastLogin says "Your username contains illegal characters" to floodgate users.

### ISSUE THREAD

**Issue by frafol** [games647/FastLogin#632]
### What happened?

FastLogin says "Your username contains illegal characters" when a user with any floodgate prefix connect to the bungeecord server.

![image](https://user-images.githubusercontent.com/67754059/137487260-be9291ba-a3f4-49f9-8cba-e7b84df64b75.png)

I have used "*, +, -, ." and no one of these works.

Without FastLogin i can join in the server.

### Steps to reproduce

1) Setup FastLoginBungee,
2) Start Floodgate and GeyserMC,
3) Insert "*" as prefix in the Floodgate's config,
4) Try to join in the server two times.

### Server log

```shell
https://hastebin.com/iwakixebun.less
```

### Plugin version

1.11-SNAPSHOT-ef6f0fc

### Platform

BungeeCord

### Relevance

- [X] I tried the latest build
- [X] I checked for existing tickets -
If there are, please vote them with a thumps reaction and not create new ones

**Comment by games647** [comment]
Yes that's valid, because if Floodgate returns that this is a floodgate player this part of the code is unreachable.

**Comment by games647** [comment]
According to the error yes.
